drivers/volume/portworx/portworx.go

```diff
@@ -1060,6 +1057,44 @@ func (d *portworx) UpgradeDriver(version string) error {
 	return nil
 }
 
+func (d *portworx) setupObjectstore(pxNode node.Node) error {
+	volDriver, err := d.getVolumeDriverByAddress(pxNode.Addresses[0])
+	if err != nil {
+		return err
+	}
+
+	name := "objVolume"
+	req := &api.VolumeCreateRequest{
+		Locator: &api.VolumeLocator{Name: name},
+		Source:  &api.Source{},
+		Spec: &api.VolumeSpec{
+			Size:    50,
```

Comment:
what size should we keep here?

Reply:
This should be an API in the volume interface. Shouldn't get called automatically. The test should call it if required.

Reply:
Actually, this is very PX specific (and will go away envtually), so shouldn't be called from the integration test itself. So this should be done before the test starts.

Reply:
removed it from torpedo